src/models/authorized_cidrs.ts

```diff
@@ -0,0 +1,46 @@
+import * as t from "io-ts";
+import { Container } from "@azure/cosmos";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { readonlySetType } from "@pagopa/ts-commons/lib/types";
+import { CosmosdbModel, CosmosResource } from "../utils/cosmosdb_model";
+import { CIDR } from "../../generated/definitions/CIDR";
+
+export const AUTHORIZED_CIDRS_COLLECTION_NAME = "subscription-cidrs";
+const AUTHORIZED_CIDRS_MODEL_PK_FIELD = "id";
+
+/** List of Authorized IP (in CIDRs format) to allow *subscription keys* use */
+export const AuthorizedCIDRs = t.interface({
```

Comment:
Why do we name the model so different than the collection? Will it make our life easier (adding a useful abstraction for developers) or harder (we have to remember that collection and model are named different)

Reply:
Right, we will refactor AuthorizedCIDRs as SubscriptionCIDRs